Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html

```diff
@@ -109,6 +109,10 @@
             <div id="pb-greetings" class="pb-hide-below-big" *ngIf="currentUser != null && hasImage">
               {{ (getGreetingMessage() | async) + ' ' + currentUser.Firstname }}!
             </div>
+            <div id="unicorn" *featureFlag="'firstApril'">
+
+              <img src="/assets/gif/RainbowCloud.gif" height=50px; />

```

Comment:
> die demo übernimmt anscheinend die Assets nicht ^^ hab ganz lieb ein gif gemalt

du hast es falsch verwiesen
```suggestion
              <img src="assets/gif/RainbowCloud.gif" height=50px; />

```